posthog/tasks/update_cache.py

```diff
@@ -149,19 +150,17 @@ def dashboard_item_update_task_params(
     return cache_key, cache_type, payload
 
 
-def _calculate_by_filter(filter: FilterType, key: str, team_id: int, cache_type: CacheType) -> List[Dict[str, Any]]:
+def _calculate_by_filter(filter: FilterType, key: str, team: Team, cache_type: CacheType) -> List[Dict[str, Any]]:
     insight_class = CACHE_TYPE_TO_INSIGHT_CLASS[cache_type]
 
     if cache_type == CacheType.PATHS:
-        result = insight_class(filter, Team(pk=team_id)).run(filter, Team(pk=team_id))
```

Comment:
@EDsCODE I think you added this `Team(pk=team_id)` workaround, which breaks actually looking up the right timezone. Any reason you did this?

Reply:
I believe this was originally added to just avoid doing another lookup of team